index.html

```diff
@@ -259,26 +259,26 @@ <h2>EdDsaSASignatureSecp256k1</h2>
       <tbody>
         <tr>
           <td>Identifiers</td>
-          <td>EdDsaSASignatureSecp256k1, EdDsaSAPublicKeySecp256k1</td>
+          <td>EcdsaSecp256k1Signature2019, EcdsaSecp256k1VerificationKey2019</td>
         </tr>
         <tr>
           <td>Status</td>
           <td>PROVISIONAL</td>
         </tr>
         <tr>
           <td>Authors</td>
-          <td>Harlan Wood, Manu Sporny</td>
+          <td>(needs authors)</td>
         </tr>
         <tr>
           <td>Specification</td>
-          <td><a href="https://w3c-dvcg.github.io/lds-koblitz2016/">Koblitz Signature Suite 2016</a></td>
+          <td>(needs spec) <a href="https://w3c-dvcg.github.io/lds-secp256k1-2019/">Secp256k1 Signature Suite 2019</a></td>
```

Comment:
So there seems to be two different issues here. This entry should be fixed to have a type of `EcdsaKoblitzSignature2016` because it already exists and the authors are correct, etc. (follow the existing https://w3c-dvcg.github.io/lds-koblitz2016/ link for the spec).

A new entry should be created for the new `EcdsaSecp256k1Signature2019`. Then I think we'd be good.

Reply:
Sounds good, I just restored the original entry for `EcdsaKoblitzSignature2016` in addition to the new `EcdsaSecp256k1Signature2019`. Could you check?

Reply:
LGTM